pkg/helm/release/manager.go

```diff
@@ -201,10 +202,11 @@ func (m manager) UpdateRelease(ctx context.Context) (*rpb.Release, *rpb.Release,
 			// log both the update and rollback errors.
 			rollbackErr := rollback.Run(m.releaseName)
 			if rollbackErr != nil {
+				//TODO: What to change this to
 				return nil, nil, fmt.Errorf("failed update (%s) and failed rollback (%s)", err, rollbackErr)
```

Comment:
```suggestion
				return nil, nil, fmt.Errorf("failed update (%w) and failed rollback (%w)", err, rollbackErr)
```

Reply:
Per [Errorf reference](https://golang.org/pkg/fmt/#Errorf), and re-iterated in this [issue](https://github.com/golang/go/issues/34060), 
**It is invalid to include more than one %w verb**, so what should the change be?

Reply:
See https://github.com/operator-framework/operator-sdk/pull/2355#discussion_r362285536.

Reply:
Done